### PR TITLE
[LUM-488] - Compute Evmos Community Tax

### PR DESCRIPTION
**Pull request by Ricardo-Remy**

### CODE REVIEW

File: src/http/controllers/dfract.controller.ts [hunk 1]
```diff
@@ -15,14 +15,14 @@ export class DfractController {
     @ApiOkResponse({ status: 200, type: AssetInfo })
     @Get('assets/latest')
     async getLatestAsset(@Req() request: ExplorerRequest): Promise<DataResponse> {
```
Comment (Segfaultd) [inline src/http/controllers/dfract.controller.ts:17]: DefaultTake

File: src/services/asset.service.ts [hunk 2]
```diff
@@ -97,10 +97,10 @@ export class AssetService {
         }
     };
 
-    fetchLatestMetrics = async (skip: number): Promise<[AssetInfo[], number]> => {
+    fetchLatestMetrics = async (take: number, page: number): Promise<[AssetInfo[], number]> => {
```
Comment (Segfaultd) [inline src/services/asset.service.ts:100]: Return raw using builtin pagination

**Review by Segfaultd — CHANGES_REQUESTED** [review]

File: src/http/controllers/dfract.controller.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { AssetService } from '@app/services';
 import { DataResponse, DataResponseMetadata, AssetInfo, AssetHistorical } from '@app/http/responses';
 import { ExplorerRequest } from '@app/utils';
 import { AssetRequest } from '@app/http/requests';
+import { DefaultTake } from '../decorators';
```
Comment (Segfaultd) [inline src/http/controllers/dfract.controller.ts]: My eyes... :D

File: src/http/controllers/dfract.controller.ts [hunk 4]
```diff
@@ -13,12 +14,19 @@ export class DfractController {
...
 
     @ApiOkResponse({ status: 200, type: AssetInfo })
+    @DefaultTake(50)
     @Get('assets/latest')
     async getLatestAsset(@Req() request: ExplorerRequest): Promise<DataResponse> {
-        const [metrics, total] = await this._assetService.fetchLatestMetrics(request.pagination.limit, request.pagination.page);
+        const [metrics, total] = await this._assetService.fetchLatestMetrics(request.pagination.skip, request.pagination.limit);
+
+        // We only keep the latest data
+        const latestMetrics = metrics.map((el) => ({
```
Comment (Segfaultd) [inline src/http/controllers/dfract.controller.ts:23]: Please use DTO ;)

**Review by Segfaultd — CHANGES_REQUESTED** [review]

File: src/http/controllers/dfract.controller.ts (hunk 3 above)
Comment (Ricardo-Remy) [reply src/http/controllers/dfract.controller.ts]: How did you catch that, I fixed it already :D in this commit https://github.com/lum-network/chain-bridge/pull/100/commits/7388f13007302c68de3575370f6868b09d6ff730

**Review by Segfaultd — APPROVED** [review]

**Pull request merged**